SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.
+
+
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+If you find a security vulnerability we would like to here from you asap.
+Keeping Openshot and our users safe is very important to us.
+
+If you do find a security vulnerability you can report it to jonathan@openshot.org.
+It should take us a day to the latest a week to get back to you. 
+If we do comfirm it to be a real ssecurity vulnerability we will get a patch out hopefully asap. 
```

Comment:
```suggestion
If we confirm the reported vulnerability exists in OpenShot, we will get a patch out in a reasonable timeframe.
```

(In an ideal world this would be a concrete timeframe, "within 30 days" or "in the next release, no later than 6 weeks" or whatever. But this is a largely-volunteer effort, and commitments like that are meaningless if there isn't someone who can **promise** they'll be held to. ...Failing that, non-committal commitments are understandable. But "hopefully asap" is not a response commitment.)